bcos-executor/src/precompiled/extension/BalancePrecompiled.h

```diff
@@ -62,6 +62,10 @@ class BalancePrecompiled : public bcos::precompiled::Precompiled
         PrecompiledExecResult::Ptr const& _callParameters, const CodecWrapper& codec,
         std::string_view accountHex);
 
+    inline std::string getContractTableName(
```

Comment:
inline is ineffective here due to the implementation being in the source file

Reply:
fixed